app/reconciler/createChangeSetFromCompareStructures.ts

```diff
@@ -0,0 +1,92 @@
+import { isEqual } from 'lodash';
+import logger from '~/lib/logger.server';
+
+import { DnsRecordType } from '@prisma/client';
+
+import type { Change } from '@aws-sdk/client-route-53';
+import type { ReconcilerCompareStructure } from './ReconcilerTypes';
+
+interface InputType {
+  dbStructure: ReconcilerCompareStructure;
+  route53Structure: ReconcilerCompareStructure;
+}
+
+const createChangeSetFromCompareStructures = ({ dbStructure, route53Structure }: InputType) => {
+  const toChange: Change[] = [];
+
+  /**
```

Comment:
Split each of these two steps out into their own functions so that it's easier to read.